apps/database/src/models/Account.ts

```diff
@@ -237,15 +237,15 @@ export class Account extends Model {
     const activePurchase = await this.$getActivePurchase();
     if (!activePurchase) return false;
 
-    if (!activePurchase.plan) {
-      throw new Error("Invariant plan not loaded");
-    }
-
     if (activePurchase.$isTrialActive()) {
       return false;
     }
 
-    return activePurchase.plan.usageBased;
+    const plan =
+      activePurchase.plan ??
```

Comment:
Incoherent if you use .? after, it should be === undefined if plan could be null.